llvm/test/Analysis/KernelInfo/openmp/amdgpu.ll

```diff
@@ -0,0 +1,217 @@
+; See ./README.md for how to maintain the LLVM IR in this test.
```

Comment:
The test case needs to be stripped.

Reply:
What does stripped mean in this context?  Removed?  Reduced?

Reply:
At least we don't need debug info?

Reply:
We need debug info for the remarks location: `remark: test.c:15:9: `

Reply:
Then we wouldn't be testing that remarks have correct source locations, right?

Reply:
Oh I c. Didn't catch that part.